packages/mui-material/src/Backdrop/Backdrop.d.ts

```diff
@@ -5,9 +5,35 @@ import { TransitionProps } from '../transitions/transition';
 import { Theme } from '../styles';
 import { BackdropClasses } from './backdropClasses';
 import { OverridableComponent, OverrideProps } from '../OverridableComponent';
+import { CreateSlotsAndSlotProps, SlotProps } from '../utils/types';
 
+export interface BackdropSlots {
+  root?: React.ElementType;
```

Comment:
Thanks for adding it! Now we have to remove the `slots` from `BackdropOwnProps`, otherwise it would be duplicated.

Let's also add a description for the root slot

```suggestion
  /**
   * The component that renders the root.
   * @default 'div'
   */
  root?: React.ElementType;
```